lib/src/authentication/create/create_account_view.dart

```diff
@@ -0,0 +1,133 @@
+import "package:flutter/foundation.dart" show defaultTargetPlatform;
+import "package:flutter/material.dart";
+import "package:flutter_gen/gen_l10n/app_localizations.dart";
+import "package:go_router/go_router.dart";
+import "package:kana_to_kanji/src/authentication/create/create_account_view_model.dart";
+import "package:kana_to_kanji/src/authentication/widgets/input_email.dart";
+import "package:kana_to_kanji/src/authentication/widgets/input_password.dart";
+import "package:kana_to_kanji/src/authentication/widgets/third_party_round_icon_button.dart";
+import "package:kana_to_kanji/src/core/widgets/app_scaffold.dart";
+import "package:kana_to_kanji/src/core/widgets/app_spacer.dart";
+import "package:stacked/stacked.dart";
+
+class CreateAccountView extends StatelessWidget {
+  static const routeName = "/authentication/link";
+
+  const CreateAccountView({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations l10n = AppLocalizations.of(context);
+
+    final Widget textDivider = Expanded(
+      child: Padding(
+        padding: const EdgeInsets.only(left: 20.0, right: 20.0),
+        child: Divider(
+          color: Theme.of(context).colorScheme.primaryContainer,
```

Comment:
Why use this colour instead of the default one? (`dividerColor`)

Reply:
Removed
It was to match with the containers border color